bin/interpreter.sh

```diff
@@ -73,6 +73,16 @@ if [[ ! -d "${ZEPPELIN_LOG_DIR}" ]]; then
   $(mkdir -p "${ZEPPELIN_LOG_DIR}")
 fi
 
+if [[ x"" == x${PYTHONPATH} ]]; then
+  export PYTHONPATH="${ZEPPELIN_HOME}/python/lib/pyspark.zip:${ZEPPELIN_HOME}/python/lib/py4j-0.8.2.1-src.zip"
+else
+  export PYTHONPATH="$PYTHONPATH${ZEPPELIN_HOME}/lib/pyspark.zip:${ZEPPELIN_HOME}/python/lib/py4j-0.8.2.1-src.zip"
+fi
+
+if [[ x"" == x${SPARK_HOME} ]]; then
+  export SPARK_HOME=${ZEPPELIN_HOME}
+fi
+
```

Comment:
How about move this conditional block to above the `if [[ x"" == x${PYTHONPATH} ]]; then ...` conditional block and use SPARK_HOME instead of ZEPPELIN_HOME for defining PYTHONPATH?

ie.
```
if [[ x"" == x${SPARK_HOME} ]]; then
  export SPARK_HOME=${ZEPPELIN_HOME}
fi

if [[ x"" == x${PYTHONPATH} ]]; then
  export PYTHONPATH="${SPARK_HOME}/python/lib/pyspark.zip:${SPARK_HOME}/python/lib/py4j-0.8.2.1-src.zip"
else
  export PYTHONPATH="$PYTHONPATH${SPARK_HOME}/lib/pyspark.zip:${SPARK_HOME}/python/lib/py4j-0.8.2.1-src.zip"
fi
```